VOM_Fortran/VOM-code/modules.f90

```diff
@@ -416,25 +435,27 @@ module vegmod
       REAL*8  :: cpccg_y                ! Annual grass water transport costs (mol/m2/y)
 
       REAL*8  :: etmt__                 ! Transpiration rate (m/s)
+      REAL*8  :: etmts__                ! Shaded transpiration rate (m/s)      
       REAL*8  :: etmt_h                 ! Hourly transpiration (m/h)
       REAL*8  :: etmt_d                 ! Daily transpiration rate (m/d)
       REAL*8  :: etmt_y                 ! Annual tree transpiration (mm/y)
-      REAL*8  :: etmg__(3,3,3)          ! Grass transpiration rate (m/s)
+      REAL*8  :: etmg__(3,3,3)            ! Grass transpiration rate (m/s)
+      REAL*8  :: etmgs__(3,3,3)           ! Shaded grass transpiration rate (m/s)      
       REAL*8  :: etmg_h                 ! Hourly grass transpiration (m/h)
       REAL*8  :: etmg_d                 ! Daily grass transpiration (m/d)
       REAL*8  :: etmg_y                 ! Annual grass transpiration (mm/y)
       REAL*8  :: etm_y                  ! Annual total transpiration (mm/y)
 
-      REAL*8  :: mqt_                   ! Tree water content (kg/m2)
-      REAL*8  :: mqtnew                 ! Tree water content in next time step (kg/m2)
-      REAL*8  :: mqtold                 ! Previous tree water content (kg/m2)
-      REAL*8  :: dmqt                   ! Rate of change in tree water content (kg/m2/s)
-      REAL*8  :: q_mqx                  ! Tree maximum water content per ground area (kg/m2)
-      REAL*8  :: mqsst_                 ! Tree water content at steady state (kg/m2)
-      REAL*8  :: mqsstmin               ! Tree water content at turgor loss point (kg/m2)
+      REAL*8  :: mqt_                   ! Tree water content (g/m2)
```

Comment:
Why the shift from kg to g? The VOM uses basic SI units throughout, so this can introduce confusion.